rwengine/src/engine/Animator.cpp

```diff
@@ -10,12 +10,13 @@
 
 #include <algorithm>
 #include <cmath>
+#include <utility>
 
-Animator::Animator(const ClumpPtr& _model) : model(_model) {
+Animator::Animator(ClumpPtr model) : _model(std::move(model)) {
```

Comment:
It should work the same. aka 1 copy

Reply:
I'd go to perfect forwarding.

Reply:
I don't think perfect forwarding is needed here as we only forward it to the local member `_model`.